tests/test_models/test_necks.py

```diff
@@ -13,6 +13,18 @@ def test_fpn():
     in_channels = [8, 16, 32, 64]
     feat_sizes = [s // 2**i for i in range(4)]  # [64, 32, 16, 8]
     out_channels = 8
+
+    # end_level=-1 is equal to end_level=3
+    FPN(in_channels=in_channels,
+        out_channels=out_channels,
+        start_level=0,
+        end_level=-1,
+        num_outs=5)
+    FPN(in_channels=in_channels,
+        out_channels=out_channels,
+        start_level=0,
+        end_level=3,
+        num_outs=5)
```

Comment:
It seems that the updates are not fully covered by unit tests. Please add more unit tests to fix the issues.